Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -796,6 +797,14 @@ void NetPlayDialog::OnMsgStopGame()
   QueueOnObject(this, [this] { UpdateDiscordPresence(); });
 }
 
+void NetPlayDialog::OnMsgPowerButton()
+{
+  if (Core::IsRunning())
+  {
+    QueueOnObject(this, [this] { UICommon::TriggerSTMPowerEvent(); });
```

Comment:
Is capturing `this` necessary here? It doesn't seem to be needed to call `TriggerSTMPowerEvent()`